docs/design/README.md

```diff
@@ -2,66 +2,65 @@
 Generally, design docs are on Google docs:
 
 ## Conventions
-* [Versioning of Constraint Templates](https://docs.google.com/document/d/1vB_2wm60WCVLXoegMrupqwqKAuW6gbwEIxg3vBQj6cs/edit)
+* [Versioning of Constraint Templates](https://docs.google.com/document/d/1Sg4HrNl9EYkNbn6wSGcyxULELrIrNbqS28djvNtHYoY/edit?usp=drive_link)
 
 ## In Development
```

Comment:
Perhaps we don't need this heading since most things are either in proposed state or implemented.